### DISCUSSION TITLE
Could this work for birds-flight data?

### DISCUSSION THREAD

**Discussion by ozkilim** [MilesCranmer/symbolic_deep_learning#10] [Ideas]
Thank you for this incredible work!

I was thinking of trying to apply this to some real bird flight data. We can model these flights with a simple [vicsek model ](https://en.wikipedia.org/wiki/Vicsek_model). Do you think it is possible in theory to re-discover this model with this method? 

All the best,
Oz

**Comment by MilesCranmer** [comment]
I think so. That shares a functional form with the force law examples, although you’d have to train it with a probabilistic loss instead of MSE.

**Reply by ozkilim** [reply]
Ok! I will give it a try. 

Thanks for the pointers.
